recipe/conda_build_config.yaml

```diff
@@ -1,3 +0,0 @@
-proj:
-    - '7.2.0'
-    - '8.0.1'
```

Comment:
Do you think there would be a benefit to build with 8.2 and 9.0 here?

Reply:
I was wondering about that, too but I don't think so.  Previously, cartopy was giving us trouble.  But it looks like it should be fine now:
https://github.com/conda-forge/cartopy-feedstock/pulls?q=is%3Apr+is%3Aclosed
It's being built with proj 9.0.1, just like I'm proposing we do with this pyproj release.  Presumably, it will be able to update to 9.1.0 once pyproj does.

But if you're aware of anything other than cartopy that needs these earlier proj, we could build with them still.

Reply:
It also wouldn't be too hard to add them later if it turns out we need them.

Reply:
We could add 9.0.1 when we migrate to 9.1.0, for example.

Reply:
@ocefpaf, @jorisvandenbossche and @mingwandroid, do you have thoughts about this?

Reply:
For posterity, I'm removing this because the original motivation for supporting these earlier versions (https://github.com/conda-forge/pyproj-feedstock/pull/96#discussion_r652099081) was for compatibility with `cartopy`.  This no longer seems to be a problem, as cartopy is staying up to date with the latest proj migrations:
https://github.com/conda-forge/cartopy-feedstock/pulls?q=is%3Apr+is%3Aclosed

Reply:
Removing it makes sense to me. As you mentioned, we can add it back in if it is needed.